src/wasm/extract.cpp

```diff
@@ -65,7 +68,11 @@ bool file_output::write(const char* data, size_t n) {
   position_ += n;
   total_written_ += n;
 
-  return r==n;
+  if(is_complete()){
```

Comment:
Missing space before and after parenthesis.
```suggestion
  if (is_complete()) {
```